ProjectSettings/ProjectSettings.asset

```diff
@@ -173,7 +174,7 @@ PlayerSettings:
   StripUnusedMeshComponents: 0
   VertexChannelCompressionMask: 214
   iPhoneSdkVersion: 989
-  iOSTargetOSVersionString: 8.0
+  iOSTargetOSVersionString: 9.0
```

Comment:
I'm confused why this file is changing so much.  I can see the iOS version change here, but there are tons of other changed apparently unrelated to the purpose of the PR being made.

Reply:
Are these changes (and the ones in `UnityConnectSettings.asset`) being made because the Unity project version has been updated?  If so, should the project version update be done in a separate PR?

Reply:
Reverted this.